src/AbstractFixer.php

```diff
@@ -45,6 +46,13 @@ public function __construct()
         }
     }
 
+    public function fix(\SplFileInfo $file, Tokens $tokens)
```

Comment:
Introducing a BC break requires a new major version and is not allowed on minor versions as per SemVer.